docs/getting-started/compose-infrastructure.md

```diff
@@ -776,15 +353,15 @@ kubectl delete pod see-db
 ```
 
 To clean up the infrastructure that was provisioned, you can delete the
-`PostgreSQLInstance`:
+`PostgreSQLInstance` XRC:
 
 ```console
 kubectl delete postgresqlinstance my-db
 ```
 
-> Don't clean up your `CompositeResourceDefinition` or `Composition` just yet if
-> you plan to continue on to the next section of the guide! We'll use them again
-> when we deploy an OAM application.
+> Don't clean up your `Configuration` just yet if you plan to continue on to the
```

Comment:
I think users have already deleted the `Configuration` at this point. Can we move this part above the deletion calls?

Reply:
@muvaf do you think so? Given that we install the `Configuration` with the Crossplane CLI, I'm not sure its even obvious how to clean up a `Configuration` at this point. Also, side benefit is `Configuration` installs almost instantly :)

Reply:
Oh OK, for some reason I thought the commands deleted the `Configuration`, too, but it seems it deletes only `postgresqlinstance`